riscv/encoding.h

```diff
@@ -4,7 +4,7 @@
 
 /*
  * This file is auto-generated by running 'make' in
- * https://github.com/riscv/riscv-opcodes (02b4866)
+ * https://github.com/riscv/riscv-opcodes (7876224)
```

Comment:
Since `encoding.h` has been updated after you created this PR, can you rebase on top of the HEAD of master and make sure the history is clean?

Reply:
Done. 7876224 is still more recent than what is currently in riscv-isa-sim's master.